grove/tests/amplification/test_grover.py

```diff
@@ -114,13 +115,16 @@ def test_optimal_grover(x_oracle):
 def test_find_bistring():
     bitstring_map = {"0": 1, "1": -1}
     builder = Grover()
-    with patch("pyquil.api.SyncConnection") as qvm:
+    with patch("pyquil.api.JobConnection") as qvm:
         expected_bitstring = [0, 1]
-        qvm.run_and_measure.return_value = [expected_bitstring, ]
+        jobresult = Mock(JobResult)
+        jobresult.is_done.return_value = True
+        jobresult.decode.return_value = ["".join([str(bit) for bit in expected_bitstring])]
+        qvm.run_and_measure.return_value = jobresult
```

Comment:
One my PR is merged this needs to be changed to be just: qvm.run_and_measure.return_value = ["".join...]